### PR TITLE
Remove invitation code logic from reflex logoutv2

### PR DESCRIPTION
**Pull request by ElijahAhianyo**

### CODE REVIEW

File: reflex/reflex.py [hunk 1]
```diff
@@ -629,12 +647,13 @@ def deployv2(
     vmtype: Optional[str] = typer.Option(
         None,
         "--vmtype",
-        help="Vm type id. Run `reflex apps vmtypes` to get options.",
+        help="Vm type id. Run reflex apps vmtypes list to get options.",
```
Comment (masenf) [inline reflex/reflex.py:650]: should leave the backticks
```suggestion
        help="Vm type id. Run `reflex apps vmtypes list` to get options.",
```

**Review by masenf — COMMENTED** [review]
why are the version checks being removed? i thought @Kastier1 just added those

**Comment by Kastier1** [comment]
Yeah please add back the version check!

**Comment by ElijahAhianyo** [comment]
> why are the version checks being removed? i thought @Kastier1 just added those

wow, that's on me for not checking the PR after pushing the changes, this should only be a one-liner. Fixing this

**Review by Kastier1 — CHANGES_REQUESTED** [review]
this change is not required

we are remove invitation codes for thew logout v2 not the v1

**Review by masenf — DISMISSED** [review]

**Review by Kastier1 — APPROVED** [review]

**Pull request merged**